src/ms-review.tex

```diff
@@ -297,48 +296,54 @@ \section{Introduction}
 
 
 %% Context
-While \fermi data as well as analysis tools have been public from the
+%% CN: I think Fermi-LAT data were available from the 2nd year on
+%% https://fermi.gsfc.nasa.gov/ssc/data/policy/
+While \fermi data and analysis tools have been public from the
 beginning, ground-based \gammaray astronomy has been historically conducted
-through experiments operated by independent collaborations. Each relying
+through experiments operated by independent collaborations, each relying
 on their own proprietary data and analysis software developed as part of the
-instrument. While the latter model has still been successful so far, it does not
+instrument. While this model has been so far successful, it does not
 permit easy combination of data from several instruments. This lack of
 interoperability currently limits the full exploitation of the
-available \gammaray data, especially because the different instruments often have
+available \gammaray data, especially in light of the fact that different instruments often have
 complementary sky coverages, and the various detection
 techniques have complementary properties in terms of energy range covered,
-duty cycle and spatial resolution. 
+duty cycle and spatial resolution.
 
 %TODO: better transition into context
+% CN: last-minute transition improvement
 The Cherenkov Telescope Array Observatory (CTAO) will be the first ground-based
 \gammaray instrument to be operated as an open observatory.
-Its high-level data\footnote{The lowest reduction level of data published by CTAO will be reconstructed event lists and corresponding instrument response functions.} will be shared publicly after
-some proprietary period, and the software required to analyze it will be distributed
-as well. To allow the re-usability of data from existing instruments and their interoperability,
-it is required to use open data formats and open tools that can support the various analysis methods
-commonly used in the field.
+Its high-level data\footnote{The lowest reduction level of data published by CTAO will be reconstructed event lists and corresponding instrument response functions.}
```

Comment:
Here I tried to convey why adapting a common data format is beneficial for current intrsuments.
@adonath, @bkhelifi, @registerrier, @AtreyeeS please take a look.